packages/plugin-hardhat/README.md

```diff
@@ -12,13 +12,25 @@ npm install --save-dev @openzeppelin/hardhat-upgrades
 npm install --save-dev @nomiclabs/hardhat-ethers ethers # peer dependencies
 ```
 
+And register the plugins in your [`hardhat.config.js`](https://hardhat.org/config/):
+
+```js
+// Javascript
+require('@nomiclabs/hardhat-ethers');
+require('@openzeppelin/hardhat-upgrades');
+
+// Typescript
+import '@nomiclabs/hardhat-ethers';
+import '@openzeppelin/hardhat-upgrades';
+```
```

Comment:
Ethers plugin is implicit.

```suggestion
And register the plugin in your [`hardhat.config.js`](https://hardhat.org/config/):

```js
// JavaScript
require('@openzeppelin/hardhat-upgrades');

// TypeScript
import '@openzeppelin/hardhat-upgrades';
```
```